contact.html

```diff
@@ -128,4 +128,23 @@ <h3 class="heading-title">Why Maturify? </h3>
 
 <script src="js/velocity.min.js"></script>
 <script src="js/ScrollMagic.min.js"></script>
-<script src="js/animation.velocity.min.js"></script> --><script src="js/combined/all.js"></script><!-- ...end JS Script --></body></html>
\ No newline at end of file
+<script src="js/animation.velocity.min.js"></script> --><script src="js/combined/all.js"></script><script>function resetSubscriptionForm(e) {
```

Comment:
check here

Reply:
This can't be changed cause it happens when the gulp buid command is run and the code is minified.